package.xml

```diff
@@ -12,6 +12,9 @@
   <depend>moveit_ros_planning_interface</depend>
   <depend>rclcpp</depend>
   <depend>benchmark</depend>
+  <depend>dynmsg</depend>
+  <depend>nav_msgs</depend>
+  <depend>ament_index_cpp</depend>
```

Comment:
Would it make sense to add the rmw implementations here as dependencies?

Reply:
It absolutely makes sense. I just missed this part. I fixed this new commit. Thank you.